File: Python/Stack_linked_list/utilities.py
def print_all(stack):
    # Start at the head
    node = stack.head
    print("[", end='')
    # Iterate through and print
    while node:
      print(node.value, end='')
      if node.next:
        print(", ", end='')
      node = node.next
    print("]")

File: C/Stack_linked_list/utilities.c
#include "stack.h"
#include <stdio.h>

void printAll(Stack *s) {
	Node *head = s->head;
	if (!head) {
		printf("empty\n");
		return;
	}
	Node *current = head;
	printf("[");
	while(current) {
		printf("%hd", current->value);
		// If not last
		if (current->next) {
			printf(", ");
		}
		current = current->next;
	}
	printf("]\n");
}
